drivers/usb/usbport/usb2.c

```diff
@@ -7,22 +7,2236 @@
 
 #include "usbport.h"
 
-//#define NDEBUG
+#define NDEBUG
 #include <debug.h>
 
+static const UCHAR CMASKS[USB2_MICROFRAMES] = {
```

Comment:
Can you explain what these are?

Reply:
See EHCI spec.: "Table 3-20. Endpoint Capabilities: Queue Head DWord 2 (cont.)". Split Completion Mask (µFrame C-Mask) field description.